### PR TITLE
)=

### PR DESCRIPTION
**Pull request by BashayrYahya**

### CODE REVIEW

**Comment by BashayrYahya** [comment]
..

File: README.md [hunk 1]
```diff
@@ -1,9 +1,99 @@
...
+var base: Double
+var height: Double
+
+      init { //  initialize the properties
+         base = 2.2
+         height = 2.2
+      }
+
+      override fun computeArea(): Double { //  override the computeArea function to use it
+         return ((height * base) / 2)
```
Comment (shuruqt1000) [inline README.md:55]: where did you use this ?

File: README.md [hunk 2]
```diff
@@ -1,9 +1,99 @@
...
+
+         init {
+            // here I initialized the properties
+            pi = 3.1415
+            radius = 2.2
+         }
+
+         //Circle area = pi times the radius squared (A = π*r²)
+         override fun computeArea():Double { // override the computeArea fun to use it
+
```
Comment (shuruqt1000) [inline README.md:87]: where did you use these value

File: README.md [hunk 3]
```diff
@@ -1,9 +1,99 @@
...
+var rec = Rectangle()
+var cir = Circle()
+
+println(tri.computeArea())
+println(rec.computeArea())
+println(cir.computeArea())
+}
+open class Shape() {  //super class
+var shapeName: String
+var shapeColor: String
```
Comment (shuruqt1000) [inline README.md:25]: why did you make this open ?

**Review by shuruqt1000 — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (BashayrYahya) [reply README.md:25]: to inherits from super class (shape)

**Comment by shuruqt1000** [comment]
complete

**Pull request closed without merging**